src/features/tokens/TokenListModal.tsx

```diff
@@ -143,12 +144,22 @@ export function TokenList({
             </div>
             <div className="ml-2 min-w-0 shrink text-left">
               <div className="w-full truncate text-xs">
-                {t.token.collateralAddressOrDenom || t.token.addressOrDenom || 'Native chain token'}
+                {(isIntentStandard(t.token.standard)
+                  ? t.token.collateralAddressOrDenom
+                  : t.token.addressOrDenom) ?? 'Native chain token'}
               </div>
               <div className="mt-0.5 flex space-x-1 text-xs">
                 <span>{`Decimals: ${t.token.decimals}`}</span>
                 <span>-</span>
                 <span>{`Chain: ${getChainDisplayName(multiProvider, t.token.chainName)}`}</span>
+                {isIntentStandard(t.token.standard) && (
+                  <>
+                    <span>-</span>
+                    <span>
+                      <i>via intents</i>
+                    </span>
+                  </>
+                )}
```

Comment:
if it's an intent, we let the user know (this is useful in the case that the same token can be bridged via WR or intents)